stdlib/src/collections/list.mojo

```diff
@@ -470,22 +470,23 @@ struct List[T: CollectionElement, hint_trivial_type: Bool = False](
     # Methods
     # ===-------------------------------------------------------------------===#
 
-    fn bytecount(self) -> Int:
-        """Gets the bytecount of the List.
+    fn byte_length(self) -> Int:
```

Comment:
**Suggestion** Split this NFC rename out to a different PR/drop it from this PR to keep things streamlined.

Reply:
done